configuration/environment_variables.rst

```diff
@@ -354,6 +354,43 @@ Symfony provides the following env var processors:
                 'trusted_hosts' => '%env(json:TRUSTED_HOSTS)%',
             ]);
 
+``env(yaml:FOO)``
+    Decodes the content of ``FOO``, which is a YAML encoded string. It returns
+    either an array or ``null``. Mostly useful in combination with
+    ``file``:
+
+        .. code-block:: yaml
+
+            # config/services.yaml
+            parameters:
+                env(SECRETS_FILE): '/opt/application/.secrets.yaml'
+                database_password: '%env(key:database_password:yaml:file:SECRETS_FILE)%'
```

Comment:
could we change the example and remove the key:database_password processor here?

Reply:
@OskarStark To make the example more concise? 

Would you ever use an array value as such?

Reply:
Hmm not sure, I haven’t any other example